packages/e2e-tests/next-app-dynamic-routes/package.json

```diff
@@ -20,7 +20,7 @@
   "homepage": "https://github.com/serverless-next.js/serverless-next.js",
   "dependencies": {
     "isomorphic-fetch": "2.2.1",
-    "next": "~10.0.1",
+    "next": "10.2.2",
```

Comment:
I wonder if it would be better to `yarn add next@latest` in the e2e-test workflow? That would test changes in nextjs automatically.

E.g. have a matrix with next@latest and next@^10.0.0 instead of the prev-* tests, then skip any tests that rely on newer features.

Reply:
Yeah, good idea. So we don't need to keep updating the versions in multiple places

Reply:
Although right now the test app states are stored in s3 based on the name of the app. Maybe we can just always run on the latest version